### ISSUE TITLE
Image viewer with scrolling

### ISSUE THREAD

**Issue by amosbird** [hzeller/timg#117]
timg is a great image `catting` tool that supports kitty and tmux. It will be perfect if we can provide a less-like viewer feature that allows to scroll around big images. In tmux we can set a huge window size to contain an wide image, but it's not easy to scroll around.

**Comment by hzeller** [comment]
Essentially allow to zoom in and then scroll around with cursor keys and such ? I have something like that on my TODO list, just need some free time :)

**Comment by amosbird** [comment]
> Essentially allow to zoom in and then scroll around with cursor keys and such

Exactly! We can put those features inside an alternative screen. It will eventually look like a terminal version of feh :)

**Comment by amosbird** [comment]
I just found that there is a relevant issue already https://github.com/hzeller/timg/issues/29
